src/calculateRentalCost.js

```diff
@@ -4,7 +4,27 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR_PER_DAY = 40;
+  const SMALL_TERM = 3;
+  const SMALL_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const RENTAL_COST = days * RENT_CAR_PER_DAY;
+
+  if (days < SMALL_TERM) {
+    return RENTAL_COST;
+  }
```

Comment:
```suggestion
  const currentPrice = days * RENT_CAR_PER_DAY;
```
Variables should be in camelCase